Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,433 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <GemCatalog/GemCatalogHeaderWidget.h>
+#include <CreateAGemScreen.h>
+#include <ProjectUtils.h>
+#include <PythonBindingsInterface.h>
+
+#include <QApplication>
+#include <QButtonGroup>
+#include <QComboBox>
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QRadioButton>
+#include <QScrollArea>
+#include <QStackedWidget>
+#include <QStyleOptionTab>
+#include <QStylePainter>
+#include <QTabBar>
+#include <QTabWidget>
+#include <QVBoxLayout>
+
+
+namespace O3DE::ProjectManager
+{
+    class TabBar : public QTabBar
+    {
+    public:
+        QSize tabSizeHint(int index) const
+        {
+            QSize s = QTabBar::tabSizeHint(index);
+            s.transpose();
+            return s;
+        }
+
+    protected:
+        void paintEvent(QPaintEvent*)
+        {
+            QStylePainter painter(this);
+            QStyleOptionTab opt;
+
+            QPoint currentTopPosition(115, 175);
+
+            QStringList strs = { "1. Gem Setup", "2. Gem Details", "3. Creator Details" };
+            for (int i = 0; i < count(); i++)
+            {
+                initStyleOption(&opt, i);
+                painter.drawControl(QStyle::CE_TabBarTabShape, opt);
+                painter.save();
```

Comment:
I know we want to replace this with stylesheets - but one thing that is missing is the current tab doesn't have a unique style - I think it's supposed to have a different background and maybe be bold?